extensions/amp-dynamic-css-classes/0.1/test/integration/test-runtime-classes.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
I don't see this notation anywhere in the codebase to put extension's integration tests in a separate folder. Is this coming from somewhere?